src/flag_gems/ops/fft.py

```diff
@@ -0,0 +1,109 @@
+import cmath
+import logging
+
+import numpy as np
+import torch
+import triton
+import triton.language as tl
+
+
+@triton.jit(do_not_specialize=["n_elements"])
+def bit_reversal_indices_kernel(
+    output_ptr,
```

Comment:
Here `output_ptr` is only used for storing the output indices. However, to generalize bit reversal to arbitrary input, I think we need to pass in both input and output which could certainly be aliased but it's fine.  That implies that the initial `indices` should be loaded to be distinguished with the tile offsets.